src/DataSource.ts

```diff
@@ -43,7 +43,7 @@ export class DataSource extends DataSourceApi<CmkQuery> {
   }
 
   getBackendType(): BackendType {
-    return this.instanceSettings.jsonData.backend ?? 'web';
+    return this.instanceSettings.jsonData.backend ?? 'rest';
```

Comment:
I'd be ok with this, but can't we provide default values for new config options somewhere else?

I'm gonna invest a few minutes into googling how other plugins do that.

Reply:
Ok, there seems to be no significantly better way to do that. Only thing I could think of would be in the Constructor of this class that I can't commen on.

But would look like this:
```
function updateSettings(stuff) {
  stuff.new_config_item = default;
}

...
  constructor(private instanceSettings: DataSourceInstanceSettings<DataSourceOptions>) {
    super(updateSettings(instanceSettings));
    this.webBackend = new WebApiBackend(this);
    this.restBackend = new RestApiBackend(this);
  }
...

A TODO is good enough here for me.